### PR TITLE
SX Tailwind: Support transitions

### PR DESCRIPTION
**Pull request by michalsanger**
This also adds support for CSS variables into SX.

### CODE REVIEW

File: src/sx/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "homepage": "https://github.com/adeira/universe/tree/master/src/sx",
   "license": "MIT",
   "private": false,
-  "version": "0.15.1",
+  "version": "0.15.2",
```
Comment (tbergq) [inline src/sx/package.json:7]: I think the removal of the CSS reset will be released when this is bumped, so better make it `0.16.0`

File: src/sx/src/transformStyleName.js [hunk 2]
```diff
@@ -3,5 +3,10 @@
 import * as changeCase from 'change-case';
 
 export default function transformStyleName(styleName: string): string {
+  if (styleName.startsWith('--')) {
+    // CSS variable
+    return styleName;
```
Comment (tbergq) [inline src/sx/src/transformStyleName.js:8]: It already works with CSS variables, what makes this change necessary?

Comment (michalsanger) [reply src/sx/src/transformStyleName.js:8]: See the `src/sx/src/__tests__/transformStyleName.test.js`. It was transforming the variable name.

Comment (tbergq) [reply src/sx/src/transformStyleName.js:8]: Aha, I see, this is for defining the variable, not using it 👍

File: src/sx/package.json (hunk 1 above)
Comment (michalsanger) [reply src/sx/package.json]: OK, done

**Review by tbergq — APPROVED** [review]

File: src/sx/package.json (hunk 1 above)
Comment (mrtnzlml) [reply src/sx/package.json]: In general, I'd say releasing patch versions for 0.x is not necessary - feel free to always bump the minor. Semver understands these versions a bit differently and more aggressively:

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

Also in FAQ:

> How should I deal with revisions in the 0.y.z initial development phase?
>
> The simplest thing to do is start your initial development release at 0.1.0 and then increment the minor version for each subsequent release.

But up to you, ofc. https://semver.org/

**Pull request merged**